lib/create-environment-api.js

```diff
@@ -831,6 +834,75 @@ export default function createEnvironmentApi ({
       .then((response) => wrapUiExtension(http, response.data), errorHandler)
   }
 
+  /**
+   * Gets an App Installation
+   * @memberof ContentfulEnvironmentAPI
+   * @param {string} id - AppDefinition ID
+   * @return {Promise<AppInstallation.AppInstallation>} Promise for an App Installation
+   * @example
+   * const contentful = require('contentful-management')
+   *
+   * const client = contentful.createClient({
+   *   accessToken: '<content_management_api_key>'
+   * })
+   *
+   * client.getSpace('<space_id>')
+   * .then((space) => space.getEnvironment('<environment-id>'))
```

Comment:
some formatting in the dev docs is not required, but nice

Reply:
@FLoppix